src/Castle.Windsor/Castle.Windsor.csproj

```diff
@@ -33,10 +33,18 @@
 		<Reference Include="System.Web" />
 	</ItemGroup>
 
-	<ItemGroup Condition="'$(TargetFramework)'=='netstandard1.6'">
-		<PackageReference Include="System.Runtime.Serialization.Formatters" Version="4.3.0" />
-		<PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
-		<PackageReference Include="System.Threading.Thread" Version="4.3.0" />
-		<PackageReference Include="Microsoft.Extensions.DependencyModel" Version="1.1.2" />
-	</ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0' OR '$(TargetFramework)'=='net5.0'">
+    <PackageReference Include="System.Runtime.Serialization.Formatters" Version="4.3.0" />
+    <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
+    <PackageReference Include="System.Threading.Thread" Version="4.3.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
+    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="2.1.0" />
+  </ItemGroup>
```

Comment:
Might as well keep them all together like they were in .NET Standard 1.6.

Reply:
You mean the version 1.1.2?

Reply:
> Might as well keep them all together like they were in .NET Standard 1.6.
i.e. in the same `ItemGroup` like you've now done.

> You mean the version 1.1.2?
2.1.0 was probably the right version to use now since moving to .NET Standard 2.0.